orchestration/task_dispatcher.py

```
@@ -2460,19 +2577,39 @@ def create_dynamic_agent(self, agent_spec: dict) -> bool:
                 if attempt_cli == "minimax":
-                    runtime_minimax_key = os.environ.get("MINIMAX_API_KEY", "")
+                    runtime_minimax_key = resolve_minimax_api_key()
                     if runtime_minimax_key:
-                        attempt_env_set_commands += f"export ANTHROPIC_API_KEY={shlex.quote(runtime_minimax_key)}\n"
-
-                # Build cleanup commands to prevent env vars from leaking between CLI attempts
-                env_cleanup_vars = ["ANTHROPIC_BASE_URL", "ANTHROPIC_MODEL", "ANTHROPIC_SMALL_FAST_MODEL", "API_TIMEOUT_MS", "CLAUDE_CODE_DISABLE_NONESSENTIAL_TRAFFIC"]
-                # Only add cleanup for non-MiniMax attempts; MiniMax sets these intentionally
+                        minimax_runtime_auth_token = runtime_minimax_key
+                        # Unset CLAUDECODE to prevent "nested session" error
+                        attempt_env_set_commands += f"unset CLAUDECODE 2>/dev/null || true\n"
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)